ngraph/core/src/op/max_pool.cpp

```diff
@@ -227,3 +319,54 @@ shared_ptr<Node> op::v8::MaxPool::clone_with_new_inputs(const OutputVector& new_
                                     m_axis,
                                     m_pads_value);
 }
+
+bool op::v8::MaxPool::has_evaluate() const {
+    NGRAPH_OP_SCOPE(v8_MaxPool_has_evaluate);
+    switch (get_input_element_type(0)) {
```

Comment:
Should I enable it for other data types as well? In particular I'm thinking about int8 and uint8